### ISSUE TITLE
https://image.online-convert.com/convert-to-bmp bmp does not display from converted png

### ISSUE THREAD

**Issue by ptorrone** [adafruit/Adafruit_CircuitPython_ImageLoad#3]
https://image.online-convert.com/convert-to-bmp bmp does not display from converted png. i used whatever the default settings are on the page, did not make any changes.

attached (files.zip) has the before.png and the after.bmp, the bmp does not display.

[files.zip](https://github.com/adafruit/Adafruit_CircuitPython_ImageLoad/files/2469851/files.zip)

![online-convert](https://user-images.githubusercontent.com/1685947/46820198-99bbc700-cd53-11e8-9fc2-608a33f8a898.jpg)

**Comment by oddstr13** [comment]
The provided output bitmap is 32bit.
```
$ python3 test_bmp.py after.bmp 
Color depth: 32
Compression: 3
Colors: 0
Is true color BMP:  True
```

One of the following imagemagick commands should in theory provide valid bitmaps:

**1-bit:**
```
$ convert before.png -compress none -type bilevel 1.bmp
$ python3 test_bmp.py 1.bmp
Color depth: 1
Compression: 0
Colors: 2
Is true color BMP:  False
```

**8-bit:**
```
$ convert before.png -compress none -type palette 8.bmp
$ python3 test_bmp.py 8.bmp
Color depth: 8
Compression: 0
Colors: 256
Is true color BMP:  False
```

**8-bit with RLE compression:**
```
$ convert before.png -compress rle -type palette 8c.bmp
$ python3 test_bmp.py 8c.bmp
Color depth: 8
Compression: 1
Colors: 256
Is true color BMP:  False
```

[Example output][examples]
[test_bmp.py][test_bmp.py]

See the following for the relevant implementation details: https://github.com/adafruit/Adafruit_CircuitPython_ImageLoad/blob/814bda6ad18214b75f397c991bed33338695f661/adafruit_imageload/bmp/__init__.py#L52-L63


[examples]: https://github.com/adafruit/Adafruit_CircuitPython_ImageLoad/files/3692736/image_load.zip
[test_bmp.py]: https://gist.github.com/oddstr13/8be1533f769379c63d1f9eaa1c0ff781


**In summary: The website you used does not (always?) produce compatible BMP files.**

**Comment by dherrada** [comment]
Only certain types of bitmaps work. I've linked to a learn guide page in the README where @FoamyGuy explains how to make those bitmaps in gimp

**Comment by adipotnis** [comment]
I used online-convert.com but I set the image to Monochrome.
